planetary-variables/land-surface-temperature/land-surface-temperature-backward-average/raw.js

```diff
@@ -0,0 +1,60 @@
+//VERSION=3
+
+// LST has two observations per day: 1h30 and 13h30 solar local time
+
+const date = "2022-12-31"; // The date for which the backward average is calculated
```

Comment:
Compared to the same script for SWC, why is this needed?

Reply:
For LST we have both daytime and nighttime measurements in the same collection and therefore need to use `mosaicking : "TILE"` rather than `ORBIT` to be able to select the data of interest. ATM it is [not possible](https://docs.sentinel-hub.com/api/latest/evalscript/v3/#collections) to extract the values for  `timeRange.from` and `timeRange.to` (defined in the body of the request) in a V3 EvalScript when using mosaicking `TILE`. This is something the SH team is now aware of, but for now, I needed to find an alternative way to fix the calculation date.

Reply:
Ah, makes sense!